From 8a20b0d8235c55edd2f3bd7883e2f0afe2edcd70 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Wed, 10 Jul 2024 16:29:25 +0100
Subject: [PATCH 1/1] data: Remove borked nvidia checks in udev rules file

We vendor enable these services in the /usr directory so is-enabled will
return disabled when they aren't symlinked to the /etc/ directory.
---
 data/61-gdm.rules.in | 7 -------
 1 file changed, 7 deletions(-)

diff --git a/data/61-gdm.rules.in b/data/61-gdm.rules.in
index aa1a0e06c..7f443cd25 100644
--- a/data/61-gdm.rules.in
+++ b/data/61-gdm.rules.in
@@ -46,12 +46,6 @@ TEST{0711}!="/usr/bin/nvidia-sleep.sh", GOTO="gdm_disable_wayland"
 TEST{0711}!="/usr/lib/systemd/system-sleep/nvidia", GOTO="gdm_disable_wayland"
 
 ENV{NVIDIA_PRESERVE_VIDEO_MEMORY_ALLOCATIONS}!="1", GOTO="gdm_disable_wayland"
-IMPORT{program}="/bin/sh -c 'echo NVIDIA_HIBERNATE=`systemctl is-enabled nvidia-hibernate`'"
-ENV{NVIDIA_HIBERNATE}!="enabled", GOTO="gdm_disable_wayland"
-IMPORT{program}="/bin/sh -c 'echo NVIDIA_RESUME=`systemctl is-enabled nvidia-resume`'"
-ENV{NVIDIA_RESUME}!="enabled", GOTO="gdm_disable_wayland"
-IMPORT{program}="/bin/sh -c 'echo NVIDIA_SUSPEND=`systemctl is-enabled nvidia-suspend`'"
-ENV{NVIDIA_SUSPEND}!="enabled", GOTO="gdm_disable_wayland"
 LABEL="gdm_nvidia_suspend_end"
 LABEL="gdm_nvidia_end"
 
@@ -118,4 +112,3 @@ RUN+="@libexecdir@/gdm-runtime-config set daemon WaylandEnable false"
 GOTO="gdm_end"
 
 LABEL="gdm_end"
-
-- 
2.45.2

